src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -370,6 +370,9 @@ void Renderer::Impl::render(const UpdateParameters& updateParameters) {
             if (bucket) {
                 sortedTilesForInsertion.emplace_back(tile);
                 tile.used = true;
+                if (!symbolLayer || parameters.mapMode == MapMode::Still) {
```

Comment:
This should be improved to exclude clipping for raster tiles as well. We could also add a virtual call to the Render*Layer object.

Reply:
Reference: https://github.com/mapbox/mapbox-gl-native/blob/master/src/mbgl/renderer/layers/render_symbol_layer.cpp#L95

Reply:
Can you add a comment here explaining why clipping is not needed for symbol layers, unless it's still mode?